server/src/models/GameState.ts

```diff
@@ -17,6 +20,11 @@ export class GameState {
         this.heroes[id] = new Hero(team)
     }
 
+    createHouses(id: string) {
```

Comment:
Was this supposed to create more than one?

Reply:
No. This'll get called once for each house.